src/chain_spec.rs

```diff
@@ -768,6 +768,30 @@ fn centrifuge_genesis(
 		aura: Default::default(),
 		democracy: Default::default(),
 		parachain_system: Default::default(),
+		bridge: centrifuge_runtime::BridgeConfig {
+			// Whitelist chains Ethereum - 0
+			chains: vec![0],
+			// Register resourceIDs
+			resources: vec![
+				// xCFG ResourceID to PalletBridge.transfer method (for incoming txs)
+				(
+					// TODO(all): Adapt these values
```

Comment:
This can remain the same, we will need to update it in the runtime manually, but this works fine for testing development flows.